frontends/verilog/verilog_frontend.cc

```diff
@@ -26,6 +26,8 @@
  *
  */
 
+#include <filesystem>
```

Comment:
This is causing the WASI build to fail, and from my understanding WASI doesn't _have_ a filesystem, or is agnostic of it; so I'm not sure if file list support makes sense on that platform and instead both the `#include` and the `parse_file_list`/`VerilogFileList` could be gated behind `#if !defined(__wasm)`.